rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -45,6 +45,12 @@ namespace rsx
 		rsx->invalid_command_interrupt_raised = true;
 	}
 
+	void trace_method(thread* rsx, u32 _reg, u32 arg)
+	{
+		// For unknown yet valid methods
+		LOG_NOTICE(RSX, "RSX method 0x%x (arg=0x%x)", _reg << 2, arg);
```

Comment:
Lower to LOG_TRACE to avoid spamming log since we still don't have a log_once function.